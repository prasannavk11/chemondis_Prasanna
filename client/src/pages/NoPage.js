import React from "react";
import { Link } from "react-router-dom";

const NoPage = () => {
  return (
    <div style={{ margin: "2% auto", width: "80%" }}>
      <Link to="/"> Go home</Link>
      <h1>404</h1>
    </div>
  );
};

export default NoPage;
