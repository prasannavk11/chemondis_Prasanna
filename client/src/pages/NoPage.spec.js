import { MemoryRouter } from "react-router-dom";
import { render } from "@testing-library/react";
import NoPage from "./NoPage";

afterEach(() => {
  jest.clearAllMocks();
  jest.resetAllMocks();
});

describe("NoPage", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<NoPage />, {
      wrapper: MemoryRouter,
    });
    expect(baseElement).toMatchSnapshot();
  });
});
