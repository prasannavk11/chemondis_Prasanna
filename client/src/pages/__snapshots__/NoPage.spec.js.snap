// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoPage should render successfully 1`] = `
<body>
  <div>
    <div
      style="margin: 2% auto; width: 80%;"
    >
      <a
        href="/"
      >
         Go home
      </a>
      <h1>
        404
      </h1>
    </div>
  </div>
</body>
`;
